packages/big-design/src/components/Worksheet/editors/SelectEditor/styled.ts

```diff
@@ -0,0 +1,31 @@
+import { theme as defaultTheme } from '@bigcommerce/big-design-theme';
+import styled, { css } from 'styled-components';
+
+export const SelectWrapper = styled.div<{ isEdited: boolean }>`
+  span {
+    background-color: ${({ theme }) => theme.colors.transparent};
+    border: none;
+    box-shadow: none;
+
+    &:hover:not([disabled]) {
+      border: none;
+    }
+  }
+
+  input {
+    font-size: ${({ theme }) => theme.typography.fontSize.small};
+    font-weight: ${({ theme }) => theme.typography.fontWeight.regular};
+
+    ${({ isEdited }) =>
+      isEdited &&
+      css`
+        background-color: ${({ theme }) => theme.colors.warning10};
+      `}
+  }
+
+  [role='option'] {
+    font-size: ${({ theme }) => theme.typography.fontSize.small};
+  }
+`;
```

Comment:
This approach of overwriting styles saved me the complexity of allowing the `Select` custom input components. Since this is the only scenario were we want a different input, it made sense to me. Thoughts?

Reply:
lgtm